python/frontend/cirrus/cirrus/GridSearch.py

```diff
@@ -52,6 +75,13 @@ def set_task_parameters(self, task, param_base=None, hyper_vars=[], hyper_params
         base_port = 1337
         index = 0
         num_machines = len(machines)
+
+        lc = boto3.client('lambda')
```

Comment:
Use better name

Reply:
Apparently that's an unused variable. Removed and fixed.